mopidy/http/actor.py

```diff
@@ -102,7 +102,7 @@ def __init__(self, config, core, sockets, apps, statics):
         self.io_loop = None
 
     def run(self):
-        self.app = tornado.web.Application(self._get_request_handlers())
+        self.app = tornado.web.Application(self._get_request_handlers(), cookie_secret=bcrypt.gensalt())
```

Comment:
Won't this give a new secret on each restart, invalidating previous cookies? We probably want to initialize a 0700 file with the salt and stick in a reasonable place and use that?

Reply:
A reasonable place would be in data_dir (https://docs.mopidy.com/en/latest/config/#confval-core/data_dir)

Reply:
Essentially we want something similar to https://github.com/mopidy/mopidy/blob/76da7a6bd0754f48ac1865e23117df7b62f324b8/mopidy/core/actor.py#L157 using https://github.com/mopidy/mopidy/blob/76da7a6bd0754f48ac1865e23117df7b62f324b8/mopidy/internal/storage.py if it makes sense to persist the value.

So is it reasonable to just have a new secret per run, or should we keep it constant?